test/fixtures/votes.yml

```diff
@@ -0,0 +1,185 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  user_id: 1
+  work_id: 1
```

Comment:
Instead of specifying relations by ID, you should use the names you assigned in the other yaml files. So this vote would look like:

```yaml
one:
  user: one
  work: one
```